sarif/build.gradle.kts

```diff
@@ -65,12 +65,36 @@ publishing {
             from(components["java"])
 
             pom {
-                developers {
+                url.set("https://github.com/JetBrains/qodana-sarif")
+
+                licenses {
+                    license {
+                        name.set("Apache-2.0")
+                        url.set("https://github.com/JetBrains/qodana-sarif/blob/main/LICENSE")
+                    }
+                }
+
+                developers {  // https://github.com/JetBrains/qodana-sarif/graphs/contributors
                     developer {
-                        id.set("alexeyAfanasiev")
+                        id.set("avafanasiev")
                         name.set("Alexey Afanasiev")
                         email.set("Alexey.Afanasiev@jetbrains.com")
                     }
+                    developer {
```

Comment:
I wonder, instead of mentioning individual contributors, shouldn't we set something like `JetBrains s.r.o` here, considering it's an official library?

Reply:
As long as there are @jetbrains.com emails, it's OK. Also, the library is not published to Maven Central yet.

Reply:
Thanks for clarifying :)